test/freenet/client/filter/ContentFilterTest.java

```diff
@@ -282,7 +282,9 @@ public void testEvilCharset() throws IOException {
 		FileOutputStream fos;
 		try {
 			ArrayBucket out = new ArrayBucket();
-			filter.readFilter(new ArrayBucket(total).getInputStream(), out.getOutputStream(), "UTF-16", null, null);
+			filter.readFilter(new ArrayBucket(total).getInputStream(), out.getOutputStream(), "UTF-16", null,
+          null,
+          null);
```

Comment:
Adding these two lines doesn’t do *anything* for readability here…

Reply:
They are an indication again that IntelliJ does not actually do AST manipulation for everything, but instead does a lot of things with plain string replacement. Otherwise it would either have all parameters on their own lines or all on the same line or a consistent spreading over two lines.